CHANGELOG.md

```diff
@@ -6,6 +6,34 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+
+- **CUMULUS-3861**
+  - Updated React 17 to React 18
+  - Updated Enzyme tests to React Testing Library
```

Comment:
Does this already cover the next item "Removed test script references..."? It's not totally necessary to list the test scripts. Changelog only needs to summarize the changes.

Reply:
@jennyhliu Removed the extra line item, as well as the listing of all the unit test scripts that were converted.